Replicated/Replicated.cpp

```diff
@@ -7,10 +7,28 @@
 
 
 Replicated::Replicated() {
+
+}
+
+void Replicated::InitLevel(){
+    std::cout << "in initintitnitntin";
+
+}
+
+int Replicated::GetCurrentLevelLen(){
+    return levelReader->GetLevelSize();
 }
 
-void Replicated::CreateObject(GameObject *g, GameWorld& world) {
 
+void Replicated::AddBlockToLevel(GameObject *g, GameWorld& world, PrimitiveGameObject* cur) {
```

Comment:
This func will work for cubes but not for capsules or spheres, this can be implemented another time!